topiary/src/configuration.rs

```diff
@@ -23,16 +24,13 @@ impl Configuration {
                 .name("predicate")
                 .expect("predicate capture group")
                 .as_str();
-            let mut arguments = capture
-                .name("arguments")
-                .expect("arguments capture group")
-                .as_str()
-                .split(' ');
-            log::info!("Predicate: {predicate} -  Arguments: {arguments:?}");
+            let argument = capture.name("arguments").map(|arg| arg.as_str());
```

Comment:
Is this just for expediency's sake, as we don't yet have any multi-argument predicates?

Reply:
Yeah! We used to split on spaces, which meant that `"   "` would be parsed as `\"` and `\"`, that's not so great in this case.